src/convertToObject.js

```diff
@@ -1,12 +1,40 @@
 'use strict';
 
+/**
+ * @param {string} sourceString
+ *
+ * @return {object}
+ */
+'use strict';
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `obj`. Instead, use more descriptive names that convey the purpose of the variable, such as `styleObject` or `cssProperties`.